### ISSUE TITLE
[3.6.1] Non admin cannot add custom avatar to group

### ISSUE THREAD

**Issue by evilrussian** [RocketChat/Rocket.Chat#18894]
### Description:

As in #18785 non admin still cannot add custom avatar to group
Then you saving avatar there is an error on server:
```
Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined
    at app/lib/server/functions/setRoomAvatar.js:34:46
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
    at packages/meteor.js:550:25
    at runWithEnvironment (packages/meteor.js:1286:24)
```

### Steps to reproduce:

1. Click channel and click Room information
2. Click edit and click upload and choose I want change avatar
3. Click save but not change avatar

### Expected behavior:

Changing avatar

### Server Setup Information:

- Version of Rocket.Chat Server: 3.6.1
- Operating System: Linux
- Deployment Method: kubernetes
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version: 2.17.11
- Operating System: MacOS

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by klepptor** [comment]
Same here, 3.6.1:

I20200914-06:27:47.337(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)

**Comment by lelvisl** [comment]
#18841

**Comment by ankar84** [comment]
@sampaiodiego @geekgonecrazy **Channel Avatars** shouldn't consider as released feature until that issue is fixed.
Indeed it's not released if only admins can change channels avatars.
But it is on [release notes](https://github.com/RocketChat/Rocket.Chat/releases/tag/3.6.0).

**Comment by Weuxel** [comment]
Also in 3.7.0

**Comment by ankar84** [comment]
I did try to change room avatar as Admin from **Admin UI - Rooms** with no luck.
Here is example of what that avatar looked like
![image](https://user-images.githubusercontent.com/4023037/95702723-cca43880-0c55-11eb-83ef-e3c02982c561.png)
Only after I was invited to that chat I successfully set avatar.

**Comment by danel1** [comment]
Same problem here with 3.6.2 and the newest desktop client for macOS.

`I20201019-08:52:58.781(0) Exception in setTimeout callback: TypeError: Cannot read property 'etag' of undefined     at app/lib/server/functions/setRoomAvatar.js:34:46     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)     at packages/meteor.js:550:25     at runWithEnvironment (packages/meteor.js:1286:24)`

**Comment by cprice-kgi** [comment]
i've uploaded a PR today that should resolve this issue:
https://github.com/RocketChat/Rocket.Chat/pull/19340

**Comment by ankar84** [comment]
Fixed on open.rocket.chat which on version 3.8-rc0

**Comment by sampaiodiego** [comment]
right.. fix on latest 3.8 release.
